go.mod

```diff
@@ -1,9 +1,19 @@
 module github.com/google/go-flow-levee
 
-go 1.14
+go 1.18
 
 require (
 	github.com/google/go-cmp v0.5.2
-	golang.org/x/tools v0.0.0-20200416214402-fc959738d646
+	golang.org/x/exp/typeparams v0.0.0-20220428152302-39d4317da171
+	golang.org/x/tools v0.1.11-0.20220504225841-45c8a7131235
 	sigs.k8s.io/yaml v1.2.0
 )
+
+// TODO: clean up indirect imports.
```

Comment:
Is this actionable?  I thought `// indirect` imports were the result of a direct import not building with a `go.mod` that details explicit dependencies, so the consumer has to choose explicit versions.  But other than removing our direct dependencies, I don't think we get to clean up the indirect ones, do we?

I think having them grouped separately is cleaned up enough.

Reply:
These indirect imports are actually produced automatically by `go mod tidy`, and I don't have a better way to handle them now.